CHANGELOG.md

```diff
@@ -1,3 +1,158 @@
+# 7 (UNRELEASED)
+
+A big shout out to all the contributors! We had 51 contributors in this release.
+
+This release is particularly large and featureful. Check out some of the
+highlights in the [news section](https://helix-editor.com/news/release-7-highlights/).
+
+As usual, the following is a summary of each of the changes since the last release.
+For the full log, check out the [git log](https://github.com/helix-editor/helix/compare/v0.6.0..v7).
```

Comment:
Oh, how come v0.6.0 becomes v7?

Reply:
@archseer and I were talking about versioning schemes and were thinking that semantic versioning doesn't work very well for applications. We were also thinking about calendar versioning (`22.03` or `22.03.20`) but tentatively went with incrementing integer like browsers. It's still up for debate though. I probably should've put a note in the matrix, huh? 😅